gomint-server/src/main/java/io/gomint/server/entity/animal/EntityCod.java

```diff
@@ -41,6 +45,30 @@ private void initEntity() {
         this.setHealth(3);
     }
 
+    @Override
+    protected void kill() {
+        super.kill();
+
+        if (deadTimer > 0) {
```

Comment:
Whats the reason behind this using the deadTimer and others using isDead()? isDead() should be sufficient here

Reply:
Nice catch, yes, it should be isDead(), I tried deadTimer before and it didn't work so I changed everything to isDead() 😛